irods_browser_app/static/js/irods.js

```diff
@@ -0,0 +1,236 @@
+$('#btn-signin-irods').on('click',function(){
+    if(sessionStorage.username)
+        $("#username").val(sessionStorage.username);
+    if(sessionStorage.password)
+        $("#password").val(sessionStorage.password);
+    if(sessionStorage.host)
+        $("#host").val(sessionStorage.host);
+    if(sessionStorage.port)
+        $("#port").val(sessionStorage.port);
+    if(sessionStorage.zone)
+        $("#zone").val(sessionStorage.zone);
```

Comment:
You might also want to do the login operation, to test that the credentials are actually valid, before saving them as we discussed.